Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -0,0 +1,2679 @@
+        demisto.results("The Instances were terminated")
+
+
+def create_volume_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    obj = vars(client._client_config)
+    kwargs = {'AvailabilityZone': args.get('availabilityZone')}
+
+    if args.get('encrypted') is not None:
+        kwargs.update({'Encrypted': True if args.get('encrypted') == 'True' else False})
+    if args.get('iops') is not None:
+        kwargs.update({'Iops': int(args.get('iops'))})
+    if args.get('kmsKeyId') is not None:
+        kwargs.update({'KmsKeyId': args.get('kmsKeyId')})
+    if args.get('size') is not None:
+        kwargs.update({'Size': int(args.get('size'))})
+    if args.get('snapshotId') is not None:
+        kwargs.update({'SnapshotId': args.get('snapshotId')})
+    if args.get('volumeType') is not None:
+        kwargs.update({'VolumeType': args.get('volumeType')})
+    if args.get('kmsKeyId') is not None:
+        kwargs.update({'KmsKeyId': args.get('kmsKeyId')})
+    if args.get('tags') is not None:
+        kwargs.update({
+            'TagSpecifications': [{
+                'ResourceType': 'volume',
+                'Tags': parse_tag_field(args.get('tags'))}]
+        })
+
+    response = client.create_volume(**kwargs)
+
+    data = ({
+        'AvailabilityZone': response['AvailabilityZone'],
+        'CreateTime': datetime.strftime(response['CreateTime'], '%Y-%m-%dT%H:%M:%SZ'),
+        'Encrypted': response['Encrypted'],
+        'Size': response['Size'],
+        'State': response['State'],
+        'VolumeId': response['VolumeId'],
+        'Iops': response['Iops'],
+        'VolumeType': response['VolumeType'],
+        'Region': obj['_user_provided_options']['region_name'],
+    })
+    if 'SnapshotId' in response:
+        data.update({'SnapshotId': response['SnapshotId']})
+    if 'KmsKeyId' in response:
+        data.update({'KmsKeyId': response['KmsKeyId']})
+    if 'Tags' in response:
+        for tag in response['Tags']:
+            data.update({
+                tag['Key']: tag['Value']
+            })
+
+    ec = {'AWS.EC2.Volumes': data}
+    human_readable = tableToMarkdown('AWS EC2 Volumes', data)
+    return_outputs(human_readable, ec)
+
+
+def attach_volume_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {
+        'Device': args.get('device'),
+        'InstanceId': args.get('instanceId'),
+        'VolumeId': args.get('volumeId'),
+    }
+    response = client.attach_volume(**kwargs)
+    data = ({
+        'AttachTime': datetime.strftime(response['AttachTime'], '%Y-%m-%dT%H:%M:%SZ'),
```

Comment:
this can throw value error. better calculate it before the append and in a try block.